docs/Demo.md

```diff
@@ -1,73 +1,181 @@
 # Pelorus Demo
+## Lead Time for Change and Deployment Frequency
 
-## Assumptions
-- Github SSH key is setup on the machine where the demo will run.
-- Ansible is installed
+![dora_lead_deployment](img/dora_metrics.png)
+
+
+### Assumptions
 - oc command line tools installed
-- Logged into OCP Cluster
+- Logged into OCP Cluster via the CLI and UI as kubeadmin
 
-## Goal
+### Goal
 
 In this demo, you will get a taste of how Pelorus captures a change going through the application's delivery cycle.
 
-1. Initializing Pelorus sets the baseline by looking at existing, stored data
-2. Running the demo script will create a new commit to go through the pipeline
+1. Initializing Pelorus sets the baseline by looking at existing stored data
+2. Create a new commit
 3. Watch as the metrics and trends change as new versions roll out
 
 Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
 
 ## Prerequisites
 
-Pelorus is up and running
+* Pelorus is installed and running
+  * Follow the steps outlined [here](https://github.com/konveyor/pelorus/blob/master/samples/todolist.md#pelorus-install-steps)
```

Comment:
We should use relative links so that this works the same way on github and readthedocs.

We should also make the link text more descriptive, rather than linking `here`. We could say `Follow the steps in the [todolist install guide](url)`